content/src/main/content/jcr_root/apps/acs-commons/rte-plugins/accordion/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    jsProcessor="[default:none,min:gcc]"
+    categories="rte.coralui2"/>
```

Comment:
Can we change this to `categories="acs-commons.rte-accordian.rte.coralui2"` and ask consumers to create a ClientLib?

```
<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
   jcr:primaryType="cq:ClientLibraryFolder"
   categories="rte.coralui2"
    dependencies="[acs-commons.rte-accordian.rte.coralui2]"/>
```

I think this JS gets auto-loaded in  ".../apps/acs-commons/rte-plugins/.content.xml → ...ns/rte-plugins/dialog-plugin/.content.xml" as well - not sure why, but GH wont let me comment on that line.

Basically, we don't want to load JS/CSS unless someone explicitly enables a feature.